container_transform/compose.py

```diff
@@ -218,7 +218,36 @@ def emit_entrypoint(self, entrypoint):
         return entrypoint
 
     def ingest_volumes_from(self, volumes_from):
-        return volumes_from
+        ingested_volumes_from = []
+        for vol in volumes_from:
+            ingested = {}
+            parts = vol.split(':')
+            rwo_value = None
+
+            assert(len(parts) <= 3)
```

Comment:
Could you add an explanation parameter to this assert so the error is a little more clear? ``assert(len(parts) <=3, "volume string '{}' has too many colons".format(vol))` or something like that?